indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisorSpec.java

```diff
@@ -163,29 +159,12 @@ public DruidMonitorSchedulerConfig getMonitorSchedulerConfig()
    * @return autoScaler, disable autoscale will return dummyAutoScaler and enable autoscale wiil return defaultAutoScaler by default.
    */
   @Override
-  @SuppressFBWarnings(value = "RV_RETURN_VALUE_IGNORED", justification = "using siwtch(String)")
-  public SupervisorTaskAutoscaler createAutoscaler(Supervisor supervisor)
-  {
-    String dataSource = getId();
-    SupervisorTaskAutoscaler autoScaler = new DummyAutoScaler(supervisor, dataSource);
-    AutoScalerConfig autoScalerConfig = ingestionSchema.getIOConfig().getAutoscalerConfig();
-
-    // kinesis'autoscalerConfig is always null for now, So that kinesis will hold a DummyAutoScaler.
-    // only SeekableStreamSupervisor is supported here.
-    if (autoScalerConfig != null
-            && autoScalerConfig.getEnableTaskAutoscaler()
-            && supervisor instanceof SeekableStreamSupervisor) {
-
-      String autoScalerStrategy = autoScalerConfig.getAutoScalerStrategy();
-
-      // will thorw 'Return value of String.hashCode() ignored : RV_RETURN_VALUE_IGNORED' just Suppress it.
-      switch (StringUtils.toLowerCase(autoScalerStrategy)) {
-        default: {
-          autoScaler = new DefaultAutoScaler(supervisor, dataSource, autoScalerConfig, this);
-        }
-      }
+  public SupervisorTaskAutoScaler createAutoscaler(Supervisor supervisor)
+  {
+    if (ingestionSchema.getIOConfig().getAutoscalerConfig() == null) {
+      return new NoopAutoScaler();
```

Comment:
I changed the logic here as
```
  public SupervisorTaskAutoScaler createAutoscaler(Supervisor supervisor)
  {
    AutoScalerConfig autoScalerConfig = ingestionSchema.getIOConfig().getAutoscalerConfig();
    if (autoScalerConfig != null && autoScalerConfig.getEnableTaskAutoScaler() && supervisor instanceof SeekableStreamSupervisor) {
      return autoScalerConfig.createAutoScaler(supervisor, this);
    }
    return new NoopTaskAutoScaler();
  }
```
Because if autoScaleConfig is not null but disabled. We can't createAutoScaler and need to return NoopTaskAutoScaler either.

Reply:
ok